cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -145,7 +144,7 @@ func init() {
 
 	// Create the provisioner: it implements the Provisioner interface expected by
 	// the controller
-	csiProvisioner := ctrl.NewCSIProvisioner(clientset, csiAPIClient, *csiEndpoint, *connectionTimeout, identity, *volumeNamePrefix, *volumeNameUUIDLength, grpcClient, snapClient)
+	csiProvisioner := ctrl.NewCSIProvisioner(clientset, csiAPIClient, *csiEndpoint, *connectionTimeout, identity, *volumeNamePrefix, *volumeNameUUIDLength, grpcClient, snapClient, provisionerName)
```

Comment:
`provisionerName == driverName`? why do we even have these two concepts then. Can we just call it `driverName` everywhere?

Reply:
I think it's mandatory for volume topology, where kubelet knows just driver name from driver registrar and scheduler knows only provider name from StorageClass. @msau42 knows details.

Reply:
Yes, they're the same.  external-provisioner originally was implemented allowing them to be different but there's no reason they need to be so we removed it in https://github.com/kubernetes-csi/external-provisioner/pull/186